bower.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "adapt-navigationLogo",
   "version": "2.1.0",
-  "framework": ">=5",
+  "framework": ">=5.20.0",
```

Comment:
With the removal of `.u-float-left` from the less this should be bumped to 5.24.4